package.json

```diff
@@ -62,7 +62,43 @@
           "when": "editorLangId == fsh"
         }
       ]
-    }
+    },
+    "problemMatchers": [
+      {
+        "owner": "vscode-language-fsh",
+        "name": "sushi",
+        "fileLocation": "absolute",
+        "pattern": [
+          {
+            "regexp": "^(warn|error) (.+)$",
+            "severity": 1,
+            "message": 2
+          },
+          {
+            "regexp": "^  File: (.+)$",
+            "file": 1
+          },
+          {
+            "regexp": "^  Line: (\\d+)( - (\\d+))?$",
+            "line": 1
```

Comment:
For multiline problems, do you think it would be useful to try and also specify an `endLine` property? I can't remember if we already talked about this.

Reply:
Mmm good question. I had tried this, and I think for this regular expression, the end line matches in group 3 of the regular expression, so I had tried adding `"endLine": 3` here, but it didn't seem to work. But then I also thought about it if it was working as I expected, and that's when I realized that if one multiline error caused an issue across a full definition that is 10+ lines long, do we really want to highlight all of those lines? So I think we talked about it and landed on just highlighting the first line. Now that you're seeing it in practice though, if it doesn't make sense to you, I can definitely look more into why `endLine` didn't seem to be working.

Reply:
Oh, actually, it looks like it does work if you also specify `column` and `endColumn`. I'm still not sure if we want multiple lines to be highlighted though. This is what it looks like with all the lines highlighted when you get an error for `__ has minimum cardinality 1 but occurs 0 time(s)`: 
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/30803904/160456088-53be4da8-6171-43dc-8ab2-ca057ba62134.png">

I think I personally still prefer the first line being highlighted, but I'd like your opinion!

Reply:
Personally, I think that's a bit much and would also prefer just highlighting the first line.

Reply:
Alrighty I'll leave it as is!